runtime/doc/options.txt

```diff
@@ -1386,6 +1386,21 @@ A jump table for the options with a short description can be found at |Q_op|.
 	This option cannot be set from a |modeline| or in the |sandbox|, for
 	security reasons.
 
+						*'completeslash'* *'csl'*
+'completeslash' 'csl'	string	(default: "")
+			local to buffer
+			{not in Vi}  {only for MS-Windows}
+	When this option is set it overrules 'shellslash' for completion:
+	- When this option is set to "slash", a forward slash is used for path
+	  completion in insert mode. This is useful when editing HTML tag, or
+	  Makefile with 'noshellslash' on Windows.
+	- When this option is set to "backslash", backslash is used. This is
+	  useful when editing a batch file with 'shellslash' set on Windows.
```

Comment:
Patch 8.2.0017, https://github.com/vim/vim/commit/6f345a14, replaces `Windows` with `MS-Windows`.

Reply:
In retrospect, I think I'll keep this out of the PR, since this looks like a good intro patch for other contributors. Also would increase the scope of the PR beyond `completeslash`.